src/test/java/org/jenkinsci/plugins/workflow/DSLTest.java

```diff
@@ -148,6 +148,30 @@ public void nonexistentFunctions() throws Exception {
         r.assertLogContains("constructible with compass and straightedge", b);
     }
 
+    /**
+     * Tests the ability to execute a step with an unnamed monomorphic describable argument.
+     */
+    @Issue("JENKINS-29711")
+    @Test
+    public void monomorphic() throws Exception {
+        WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "mon");
+        p.setDefinition(new CpsFlowDefinition("monomorphStep([firstArg:'one', secondArg:'two'])", true));
```

Comment:
And no, no pre-existing tests from before this PR failed due to that change not being there...because we weren't running any tests against this use case! Now we are in the tests I've added in this PR. Ergo.